catroidTest/src/org/catrobat/catroid/test/physics/PhysicsSpriteCloneTest.java

```diff
@@ -188,7 +187,7 @@ public void testSpriteClonePhysicsLookAndPhysicsObject() {
 		sprite.addScript(startScript);
 
 		PhysicsWorld physicsWorld = project.getPhysicsWorld();
-		sprite.look = new PhysicsLook(sprite, physicsWorld);
+		sprite.look = new Look(sprite);
```

Comment:
A test and also the code for cloning a physics look is missing I guess? See https://github.com/Catrobat/Catroid/blob/feature/physics/catroid/src/org/catrobat/catroid/content/Look.java#L103-L113

If so, I'll write a jira ticket :wink: